modules/atomic-widgets/styles/style-schema.php

```diff
@@ -140,6 +140,18 @@ private static function get_typography_props() {
 				'revert',
 				'revert-layer',
 			]),
+			'cursor' => String_Prop_Type::make()->enum([
+				'auto',
+				'default',
+				'none',
+				'help',
+				'pointer',
+				'progress',
+				'wait',
+				'cell',
+				'crosshair',
+				'text',
```

Comment:
I think that foe this moment we can add only the `pointer` value, and add more values as we keep on going.